Containerfile-fedora-amd64

```diff
@@ -1,15 +1,17 @@
-FROM quay.io/fedora/fedora-minimal:40 as builder
-ARG name=Fedora
 ARG version=40
+
+FROM quay.io/fedora/fedora-minimal:$version as builder
+ARG name=Fedora
 ARG arch=x64
 ARG baseurl=https://dl.fedoraproject.org/pub/fedora/linux
 WORKDIR /root
```

Comment:
Would it make sense to use `/work` to avoid it overlapping with root's home directory (which may end up writing dot files or caches)?